context/app/static/js/components/Detail/provenance/ProvTableTile/ProvTableTile.jsx

```diff
@@ -30,10 +34,10 @@ function ProvTableTile(props) {
           id={id}
           invertColors={isCurrentEntity}
           entityData={entityData}
-          descendantCounts={displayDescendantCounts}
+          descendantCounts={descendantCountsToDisplay}
         />
       )}
-      {isLastTile && entity_type !== 'Donor' && allDescendantCounts[entity_type] > 0 && (
+      {isLastTile && entity_type !== 'Donor' && descendantCounts?.[entity_type] > 0 && (
```

Comment:
Again, I think this is fine in this case, but I'd be careful about the idiom... It might be safer to pull out the key check. What I worry about is someone refactoring this down the road, perhaps changing the direction of the comparison, and being surprised because any comparison with `undefined` is false.